src/components/20-molecules/datepicker/index.react.js

```diff
@@ -0,0 +1,7 @@
+import withReact from '../../../utils/with-react';
+import AXADatepicker from './index';
+
+export default createElement => props =>
+  withReact(createElement)(AXADatepicker.tagName, {
+    ...props,
```

Comment:
dont use deconstructor here, we agreed with @MKaHead that we explicitly whitelist props not pass all from react to CE

Reply:
Fixed. Is there a specific reason we do that?

Reply:
I think so that not all props from the react comp go on the CE but only those that are whitelisted.. like a interface that is strict and protected. but i did the same changes as u did and talked to marco and he gave me another explanation that i cant remeber but made sense :)